lib/commands/init.js

```diff
@@ -26,7 +26,7 @@ class Init extends BaseCommand {
 
   static name = 'init'
   static usage = [
-    '<package-spec> (same as `npx <package-spec>)',
+    '<package-spec> (same as `npx <package-spec>`)',
```

Comment:
You will need to run `npm run snap` in order to update the snapshots that capture usage output.

You may also have to run `make freshdocs`